jep/306/README.adoc

```diff
@@ -34,12 +34,14 @@ endif::[]
 //
 // Uncomment if there is an associated placeholder JIRA issue.
 | JIRA
-| https://issues.jenkins-ci.org/browse/JENKINS-50294[JENKINS-50294]
+| link:https://issues.jenkins-ci.org/browse/JENKINS-50294[JENKINS-50294]
```

Comment:
Aren't these redundant? On the jenkins.io site, URLs with protocol are automatically links.